OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/main.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
```

Comment:
It's bad practice to use unversioned Python. Please change all instances to `python3`.